src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -1828,7 +1830,7 @@ bool P104_data_struct::webform_load(struct EventStruct *event) {
       , F("Vertical")
     # endif   // ifdef P104_USE_VERTICAL_FONT
     # ifdef P104_USE_EXT_ASCII_FONT
-      , F("Ext. ASCII")
+      , F("Extended ASCII")
```

Comment:
Too bad "extended ASCII" is not as standard as you might expect.
It is code page dependent and I'm sure it doesn't use the same character set as I used to know from memory when I was using the MSX...

Reply:
> Too bad "extended ASCII" is not as standard as you might expect.

The only 'extension' I'm expecting here is that it's extended compared to the Default font 😉 
Probably have to create tables with the characters used for all the fonts, that's quite a bit of work, I guess 🤔

Reply:
And also adding a lot to the bin size.
I guess you may get a request for "Polish" characters though ;)

Reply:
> adding a lot to the bin size

That is why in the ESP8266 `Display` build all but the Default and Numeric Double-height fonts are excluded, else it wouldn't fit 😞 

> you may get a request for "Polish" characters

I hope these are in the Extended ASCII set, though I didn't yet get a good look at it. Maybe this should then also be included in that ESP6266 Display build, recent build-size reductions have created a bit of room here.